src/Azure.Deployments.Extensibility.Providers.Graph/GraphHttpClient.cs

```diff
@@ -0,0 +1,60 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Net.Http.Headers;
+using System.Text;
+using System.Text.Json.Nodes;
+using System.Threading.Tasks;
+
+namespace Azure.Deployments.Extensibility.Providers.Graph
+{
+    public class GraphHttpClient
+    {
+        private HttpClient _httpClient;
+        private string _baseUri;
+
+        public GraphHttpClient()
+        {
+            _httpClient = new HttpClient();
```

Comment:
We should actually use the MS Graph Core SDK http client instead of a native http client: https://github.com/microsoftgraph/msgraph-sdk-dotnet-core